react/Spinner/index.jsx

```diff
@@ -13,19 +14,44 @@ export const Spinner = ({
   size,
   className
 }) => {
+  let realSize = ''
+  switch (size) {
+    case 'tiny':
+      realSize = 8
+      break
+    case 'small':
+      realSize = 12
+      break
+    case 'medium':
+      realSize = 16
+      break
+    case 'large':
+      realSize = 24
+      break
+    case 'xlarge':
+      realSize = 36
+      break
+    case 'xxlarge':
+      realSize = 80
+      break
+  }
   return (
     <div
       className={cx(
         styles['c-spinner'],
         {
           [styles['c-spinner--middle']]: middle,
-          [styles['c-spinner--nomargin']]: noMargin,
-          [styles[`c-spinner--${color}`]]: color,
-          [styles[`c-spinner--${size}`]]: size
+          [styles['c-spinner--nomargin']]: noMargin
         },
         className
       )}
     >
+      <Icon
+        icon="spinner"
+        color={color}
+        className={iconStyle['icon--spin']}
```

Comment:
`<Icon />` has a props `spin` that already add the class.
See https://github.com/cozy/cozy-ui/blob/master/react/Icon/index.jsx#L45-L47
Unless you had an issue with this but it should use the props, right?

Reply:
thanls!